utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/confluent/UniversalSchemaSupport.scala

```diff
@@ -31,7 +31,7 @@ trait UniversalSchemaSupport {
   def typeDefinition(schema: ParsedSchema): TypingResult
   def extractSinkValueParameter(schema: ParsedSchema)(implicit nodeId: NodeId): ValidatedNel[ProcessCompilationError, SinkValueParameter]
   def sinkValueEncoder(schema: ParsedSchema, mode: ValidationMode): Any => AnyRef
-  def validateRawOutput(schema: ParsedSchema, t: TypingResult, mode: ValidationMode): ValidatedNel[OutputValidatorError, Unit]
+  def parameterValidator(schema: ParsedSchema, mode: ValidationMode): SinkValueValidator
```

Comment:
Why we need this method if we already have `SinkValueParameter` which takes responsibility of the validation of this nested fields. Why not to do it in `extractSinkValueParameter`. Let's keep this class simple because it will be hard to add new schema supports incrementally.